x-pack/plugins/apm/ftr_e2e/cypress/integration/read_only_user/service_overview/service_overview.spec.ts

```diff
@@ -32,66 +97,167 @@ describe('Service Overview', () => {
     await synthtrace.clean();
   });
 
-  beforeEach(() => {
-    cy.loginAsReadOnlyUser();
+  describe('renders', () => {
+    before(() => {
+      cy.loginAsReadOnlyUser();
+      cy.visit(baseUrl);
+    });
+    it('transaction latency chart', () => {
+      cy.get('[data-test-subj="latencyChart"]');
+    });
+
+    it('throughput chart', () => {
+      cy.get('[data-test-subj="throughput"]');
+    });
+
+    it('transactions group table', () => {
+      cy.get('[data-test-subj="transactionsGroupTable"]');
+    });
+
+    it('error table', () => {
+      cy.get('[data-test-subj="serviceOverviewErrorsTable"]');
+    });
+
+    it('dependencies table', () => {
+      cy.get('[data-test-subj="dependenciesTable"]');
+    });
+
+    it('instances latency distribution chart', () => {
+      cy.get('[data-test-subj="instancesLatencyDistribution"]');
+    });
+
+    it('instances table', () => {
+      cy.get('[data-test-subj="serviceOverviewInstancesTable"]');
+    });
   });
 
-  it('persists transaction type selected when clicking on Transactions tab', () => {
-    cy.visit(baseUrl);
-    cy.get('[data-test-subj="headerFilterTransactionType"]').should(
-      'have.value',
-      'request'
-    );
-    cy.get('[data-test-subj="headerFilterTransactionType"]').select('Worker');
-    cy.get('[data-test-subj="headerFilterTransactionType"]').should(
-      'have.value',
-      'Worker'
-    );
-    cy.contains('Transactions').click();
-    cy.get('[data-test-subj="headerFilterTransactionType"]').should(
-      'have.value',
-      'Worker'
-    );
+  describe('transactions', () => {
```

Comment:
I kept this tests for the transactions because they were already there, but I think we need to create a specific test file for the transactions flow
What do you think @cauemarcondes @gbamparop ?

Reply:
Or test all the tabs flow here

Reply:
From what I can see, where are testing if the transaction type is persisted when navigating from service overview to something else, so IMO it makes sense to keep it here. What you could do is to improve the `describe` title.